.papr.sh

```diff
@@ -55,6 +55,7 @@ export PATH=$HOME/gopath/bin:$PATH
 export GOSRC=/$GOPATH/src/github.com/projectatomic/libpod
 
 ${PACKAGER} install -y \
```

Comment:
might wanna install the [centos-release-container](http://cbs.centos.org/kojifiles/packages/centos-release-container/1/6.el7.centos/noarch/centos-release-container-1-6.el7.centos.noarch.rpm) package for CentOS envs before searching for cri-o rpm